app/Helper/Runner.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace ArtifactCreation\Helper;
+
+
+use ArtifactCreation\Model\ConfigurationAbstract;
+
+/**
+ * Main runner for commands passed by configuration objects
+ *
+ * Class Runner
+ * @package ArtifactCreation\Helper
+ */
+class Runner
+{
+    /**
+     * Execute a command using the passed configuration
+     *
+     * @param ConfigurationAbstract $configuration
+     */
+    public function execute(ConfigurationAbstract $configuration)
+    {
+        $fullCommand = sprintf(
```

Comment:
If would be shell_exec [caused with security issues in some cases] used  then we should validate / escape / sanitize  string which would be used as part of shell command.